### DISCUSSION TITLE
Why there is a black line in the sagittal view of  TIGRE reconstruction?

### DISCUSSION THREAD

**Discussion by zhou9240** [CERN/TIGRE#586] [Q&A]
![image](https://github.com/user-attachments/assets/fa1520a7-051f-4b77-9d72-4346f063faa2)

As shown in the above figure, there is a horizontal line in the sagittal view of TIGRE reconstruction. I didn't find any problems in the projection images.  Would you please tell me what may be the reason?  

Thanks
Zhongxing

**Comment by AnderBiguri** [comment]
thats strange indeed. Can you share the exact geometry that produced this?
